app/views/posts/index.haml

```diff
@@ -0,0 +1,9 @@
+%h1 Posts
+- @posts.each do |post|
+  %h3= post.title
+  %p Created #{time_ago_in_words(post.created_at)} ago.
```

Comment:
such logic in views is a very bad practice. Later on I'll show you how to refactor it using decorators.

Reply:
As for now we're living it this way?

Reply:
yes